e2e/cypress/integration/messaging/ctrl_cmd_k_open_dm_with_mouse_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***********************************************************  ****
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let firstUser;
+    let secondUser;
+
+    before(() => {
+        // # Login as test user
+        cy.apiInitSetup().then(({team, user}) => {
+            firstUser = user;
+            testTeam = team;
+
+            // # Create a second user that will be searched
+            cy.apiCreateUser().then(({user: user1}) => {
+                secondUser = user1;
+                cy.apiAddUserToTeam(testTeam.id, secondUser.id);
+            });
+
+            cy.apiLogin(firstUser);
+
+            // # Visit created test team
+            cy.visit(`/${testTeam.name}`);
+        });
+    });
+
+    it('T3294 - CTRL/CMD+K - Open DM using mouse', () => {
```

Comment:
```suggestion
    it('MM-T3294 - CTRL/CMD+K - Open DM using mouse', () => {
```

Reply:
Lol, my bad 😄  That's the correct one @josephk96.